scripts/generate.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# This script generates all output data from the input files in this repository.
+#
+# If executed without arguments, it downloads the required external files and
+# executes all generatrors:
+#
+#     $ scripts/generate.sh
+#
+# The `fetch` and `generate` sub-commands can be used to execute individual
+# stages:
+#
+#     $ scripts/generate.sh fetch
+#     $ scripts/generate.sh generate nace
+set -euo pipefail
+
+GENERATORS=(
+    combined_nomenclature
+    custom_products
+    envo
+    model_terms
+    nace
+    open_energy_ontology
+    qudt
+    supplements
+)
+
+declare -A GENERATOR_ARGS
+GENERATOR_ARGS=(
+    [combined_nomenclature]=sentier_vocab/CN_2024.rdf
+    [nace]=sentier_vocab/CN_2024.rdf
+)
+
+main() {
+    [[ "$#" -eq 0 ]] && set -- all
+    local cmd=$1; shift
+    case "$cmd" in
+    all) fetch; generate;;
+    fetch) fetch "$@";;
+    generate) generate "$@";;
+    *) usage;;
+    esac
+}
+
+usage() {
+    echo >&2 <<EOF
+Usage: $0 [CMD [ARG...]]
+
+Commands:
+
+    [all]
+    fetch
+    generate [MODULE...]
+EOF
+    return 1
+}
+
+fetch() {
```

Comment:
You found the magic way to get through their obfuscation. Awesome!

Reply:
haha, yes, I gave myself 15m to try to bypass it, and made it in the end.

It's one of the most convoluted way to download a public file I've seen.